package.json

```diff
@@ -8,13 +8,16 @@
   "main": "index.js",
   "scripts": {
     "geolite": "node bin/getmaxmind.js",
-    "start": "node-lambda run -j test/support/test-event.json",
+    "start": "npm run start-bigquery && npm run start-pingbacks",
+    "start-bigquery": "PINGBACKS=0 node test/integration/runner.js",
+    "start-pingbacks": "PINGBACKS=1 node test/integration/runner.js",
     "test": "mocha test/",
     "ci": "istanbul cover _mocha -- test/ && codecov",
-    "watch": "mocha test/ -w",
-    "deploy-dev": "node bin/deploy development",
-    "deploy-staging": "node bin/deploy staging",
-    "deploy-prod": "node bin/deploy production"
+    "build": "npm run build-sync && npm run build-prune && npm run build-zip",
```

Comment:
I sort of expect all these build commands to get copy/pasted between the lambda repos.